tsconfig.json

```diff
@@ -48,7 +48,7 @@
 
     /* Emit */
     "declaration": true /* Generate .d.ts files from TypeScript and JavaScript files in your project. */,
-    // "declarationMap": true,                           /* Create sourcemaps for d.ts files. */
+    "declarationMap": true /* Create sourcemaps for d.ts files. */,
```

Comment:
This setting will be ignored by `tsup`, the `.d.ts` bundling is done by `rollup-plugin-dts` which does not support declaration mapping ([link]( https://github.com/egoist/tsup/issues/311)).